frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/StatusIcon.tsx

```diff
@@ -11,17 +12,25 @@ import {
 } from '@patternfly/react-icons';
 import { getRunStatusColor, runStatus } from '../../../../utils/pipeline-augment';
 
+import './StatusIcon.scss';
+
 interface StatusIconProps {
   status: string;
   height?: number;
   width?: number;
+  shiftOrigin?: boolean;
 }
 
-export const StatusIcon: React.FC<StatusIconProps> = ({ status, ...props }) => {
+export const StatusIcon: React.FC<StatusIconProps> = ({ status, shiftOrigin, ...props }) => {
   switch (status) {
     case runStatus['In Progress']:
     case runStatus.Running:
-      return <SyncAltIcon {...props} className="fa-spin" />;
+      return (
+        <SyncAltIcon
+          {...props}
+          className={cx('fa-spin', { 'pipeline-status-icon__spin': shiftOrigin })}
```

Comment:
```suggestion
          className={cx('fa-spin', { 'pipeline-status-icon--spin': shiftOrigin })}
```

BEM is 
 - Block (`pipeline-status-icon`)
 - Element (`pipeline-status-icon__someWrapper`) -- kinda like a sub Block
 - Modifier (`pipeline-status-icon--spin`) -- when you want it to be "modified" based on a prop

Reply:
Updated.